types/src/transaction/transaction_v1/userland_transaction_v1.rs

```diff
@@ -15,6 +15,9 @@ use super::TransactionV1;
 #[cfg(any(feature = "testing", test))]
 use crate::testing::TestRng;
 use crate::{
+    addressable_entity::{
```

Comment:
userland transaction? I don't understand the distinction...all transactions originate from userland.

Reply:
That was the name I thought we agreed on in the design meeting: the two main variants of `TransactionV1Kind` being native or userland.  Happy to rename userland if you have a different suggestion.